README.md

```diff
@@ -146,6 +147,35 @@ This will store images by default in your document folder, however you may also
 :imagesdir: media
 ```
 
+### Use asciidoctor.js extensions in workspace
+
+This extension supports registration of asciidoctor.js extensions.
+To see the rendered result in the preview, the extension can use asciidoctor.js extensions while rendering it.
+
+To create an autoloadable extension you should do the next steps:
+
+* Enable the feature by `Change AsciiDoc extension scripts Security Settings` command.
```

Comment:
I couldn't find this command because this command is only visible when you have an AsciiDoc file opened.
So I think it should be an extension settings rather than a command:

![image](https://user-images.githubusercontent.com/333276/173193951-af1850c6-1de0-4b8c-93ed-25646b5c79df.png)